go.mod

```diff
@@ -16,7 +16,13 @@ require (
 	github.com/stretchr/testify v1.2.2
 	github.com/tendermint/btcd v0.0.0-20180816174608-e5840949ff4f
 	github.com/tendermint/ed25519 v0.0.0-20171027050219-d8387025d2b9 // indirect
+	github.com/tendermint/tendermint v0.30.1
+	github.com/zondax/hid v0.9.0 // indirect
+	github.com/zondax/ledger-cosmos-go v0.9.9
+	github.com/zondax/ledger-go v0.9.0 // indirect
 	golang.org/x/crypto v0.0.0-20181112202954-3d3f9f413869
 	golang.org/x/sys v0.0.0-20181116161606-93218def8b18 // indirect
 	gopkg.in/resty.v1 v1.10.3
 )
+
+replace github.com/zondax/ledger-cosmos-go => github.com/binance-chain/ledger-cosmos-go v0.9.9-binance.1
```

Comment:
emphasize this to notice user to replace this repo in their go.mod  in Readme.md please